08-components-lab/src/01-timeconverter.jsx

```diff
@@ -8,5 +8,72 @@
 
 import React from 'react';
 
-export default React.createClass({
-});
+export default class TimeConverter extends React.Component {
+    static propTypes = {
+        initialHours: React.PropTypes.number,
+        initialMinutes: React.PropTypes.number,
+        initialSeconds: React.PropTypes.number,
+    };
+
+    static defaultProps = {
+        initialHours: 0,
+        initialMinutes: 0,
+        initialSeconds: 0,
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            hours: props.initialCount,
```

Comment:
You don't really need 3 variables in state. After all hours and minutes are derived from seconds. It's enough to update only seconds with correct values